grails-app/controllers/org/broadinstitute/orsp/UrlMappings.groovy

```diff
@@ -10,6 +10,9 @@ class UrlMappings {
         }
         // TODO: Would prefer a better way to redirect a root url to a controller regardless of action.
         // Without the extra declarations, the default index action is always the one called even if /funding is hit.
+
+        '/api/addComment'(controller: 'comments', action: 'saveNewComment', method: 'POST')
+        '/api/commentsList'(controller: 'comments', action: 'getComments', method: 'GET')
```

Comment:
The comments above this line actually refer to line 17 in this file.

Reply:
Fixed.